### PR TITLE
Add ETI post-doc add

### PR DESCRIPTION
**Pull request by gonuke**
Adding information about new post-doc and ra positions for NNSA consortia

### CODE REVIEW

File: source/jobs/ra-2019-eti.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+and data streams.
+
+Experience and Desired Skills
+------------------------------
+
+Candidates will be evaluated aginst the following criteria:
+
+* Undergraduate experience in Nuclear Engineering, Computer Science,
+  Statistics,  Electrical Engineering, or relevant other field
+* Experience with machine learning, data fusion and data analytics
```
Comment (kkiesling) [inline source/jobs/ra-2019-eti.rst]: Do you actually expect students to start grad school with this experience? (can't it be learned as a grad student in classes?)

File: source/jobs/post-doc-2019-mtv.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* Author publications for refereed journals and important conference proceedings
+    
+
+| **Salary:** $50,000-70,000 depending on background and experience
+| **Start:** ASAP
+
+U.S. citizenship or permanent residency status is required by the program
+sponsor.
+
+To apply, please send a application materials to Paul Wilson at paul.wilson@wisc.edu
```
Comment (kkiesling) [inline source/jobs/post-doc-2019-mtv.rst]: "application materials" not listed for this position.

File: source/jobs/ra-2019-mtv.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+models will be improved to increase the reliability of simulating those
+measurable quantities.
+xs
+Experience and Desired Skills
+------------------------------
+
+Candidates will be evaluated aginst the following criteria:
+
+* Undergraduate experience in Nuclear Engineering, Physics, or relevant other field
+* Experience with modeling and simulation of nuclear fuel cycle facilities
```
Comment (kkiesling) [inline source/jobs/ra-2019-mtv.rst]: same question: do you actually expect an incoming grad student to have this experience when they arrive? or can this be learned?

File: source/jobs/ra-2019-eti.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    
+CNERG is seeking a qualified individual to assist in the application of modern
+data analytics algorithms to nuclear non-proliferation data sets, through the
+development of a framework that integrates both new and existing algorithms
+and data streams.
+
+Experience and Desired Skills
+------------------------------
+
+Candidates will be evaluated aginst the following criteria:
```
Comment (ngranda) [inline source/jobs/ra-2019-eti.rst]: aginst --> against

**Pull request merged**